interactions/models/internal/command.py

```diff
@@ -195,7 +198,11 @@ def param_config(self, annotation: Any, name: str) -> Tuple[Callable, Optional[d
         return (annotation, getattr(annotation, name, None))
 
     async def call_callback(self, callback: Callable, context: "BaseContext") -> None:
-        await self.call_with_binding(callback, context, **context.kwargs)  # type: ignore
+        val = await self.call_with_binding(callback, context, **context.kwargs)  # type: ignore
+        if isinstance(val, str) and isinstance(context, SendMixin):
```

Comment:
So, crazy idea – what if, on top of allowing string and embed returns, we allow for numerous embeds and/or files to be returned for interaction responses as well? This could also be really useful for one-line lambda-esque returns, however, I can see the cons being presented for this.

What would stop us from doing this instead of telling users to use `.send()` instead?